modeling-cmds/src/def_enum.rs

```diff
@@ -949,6 +949,8 @@ define_modeling_cmd_enum! {
             pub height: u32,
             /// Frames per second.
             pub fps: u32,
+            /// Video bitrate (CBR)
```

Comment:
Guideline for general writing is not to use acronyms until they've been defined
```suggestion
            /// Video feed's constant bitrate (CBR)
```

Reply:
see the engine PR on this.  and my notes about nvenc.  Default will need to be None for the time being,

Reply:
after digging for like 5 mins, I can't find my comment on that PR for the live of me.  but the gist of it was nvenc defaults the bitrate to 0 and doesn't really let us know what it's picking.  So mapping None to 0 is what i've done to re-obtain default behavior.